src/Discord.Net.Core/Entities/Channels/IGuildChannelCategory.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord
+{
+    public interface IGuildChannelCategory : IGuildChannel
```

Comment:
Should just be IChannelCategory, RestChannelCategory, etc. Current name is redundant.